### PR TITLE
Misc documentation updates

### PR DESCRIPTION
**Pull request by alex-ketch**
- Update minor dependencies
- Update README with component development and usage
- Storybook: Remove usage of deprecated Knobs Addob
- Brand: Remove corrupted image files

### CODE REVIEW

File: packages/components/readme.md [hunk 1]
```diff
@@ -17,20 +17,36 @@ Please follow the pattern of `<stencila-(component-type)>`, for example `<stenci
 
 ## Using these components
 
-### Script tag
+### Add to your project
 
-- [Publish to NPM](https://docs.npmjs.com/getting-started/publishing-npm-packages)
-- Put a script tag similar to this `<script src='https://unpkg.com/@stencila/components@0.0.1/dist/mycomponent.js'></script>` in the head of your index.html
-- Then you can use the element anywhere in your template, JSX, html etc
+There are two ways to load these components into your project, a `script` tag in an HTML file, or as an `npm` pacakge.
```
Comment (nokome) [inline packages/components/readme.md:22]:
```suggestion
There are two ways to load these components into your project, a `script` tag in an HTML file, or as a Node module.
```

File: packages/components/readme.md [hunk 2]
```diff
@@ -17,20 +17,36 @@ Please follow the pattern of `<stencila-(component-type)>`, for example `<stenci
...
-- Put a script tag similar to this `<script src='node_modules/@stencila/components/dist/mycomponent.js'></script>` in the head of your index.html
-- Then you can use the element anywhere in your template, JSX, html etc
+- Put the following script tags in the head of your HTML document:
+
+```js
+<script type="module" src="https://unpkg.com/@stencila/components@latest/dist/stencila-components/stencila-components.esm.js"></script>
+<script type="text/javascript" nomodule="true" src="https://unpkg.com/@stencila/components@latest/dist/stencila-components/stencila-components.js"></script>
+```
+
+- Then you can use the element anywhere in your template, JSX, html, etc.
```
Comment (nokome) [inline packages/components/readme.md:33]:
```suggestion
- Then you can use the element anywhere in your HTML, HTML template, JSX, etc.
```

File: packages/components/readme.md [hunk 3]
```diff
@@ -17,20 +17,36 @@ Please follow the pattern of `<stencila-(component-type)>`, for example `<stenci
...
 
-### In a stencil-starter app
+For modern browser support the `module` attribute, only the components used on the page will be requested and lazy-loaded, keeping the page size low.
```
Comment (nokome) [inline packages/components/readme.md:35]:
```suggestion
In modern browsers supporting the `module` attribute, only the components used on the page will be requested and lazy-loaded, keeping the page size low.
```

File: packages/components/readme.md [hunk 4]
```diff
@@ -17,20 +17,36 @@ Please follow the pattern of `<stencila-(component-type)>`, for example `<stenci
...
+
+#### Node Modules
```
Comment (nokome) [inline packages/components/readme.md:37]:
```suggestion
#### Node module
```

File: packages/components/readme.md [hunk 5]
```diff
@@ -17,20 +17,36 @@ Please follow the pattern of `<stencila-(component-type)>`, for example `<stenci
...
 
 - Run `npm install @stencila/components --save`
-- Add an import to the npm packages `import @stencila/components;`
-- Then you can use the element anywhere in your template, JSX, html etc
+- Import the package in your main/top-level file: `import '@stencila/components'`
+- Stencil will automatically setup the lazy-loading capabilities for the Stencil library. Then you can use the element anywhere in your template, JSX, html etc.
```
Comment (nokome) [inline packages/components/readme.md:41]:
```suggestion
- Stencil will automatically setup the lazy-loading capabilities for the Stencil library. Then you can use the element anywhere in your template, JSX, HTML etc.
```

File: packages/components/readme.md [hunk 6]
```diff
@@ -17,20 +17,36 @@ Please follow the pattern of `<stencila-(component-type)>`, for example `<stenci
...
+
+### Using and customizing components
+
+The best reference for how to use and combine these components is our [component library](https://stencila.github.io/designa/?path=/story/schema-nodes-code-chunk--with-execute-handler).
```
Comment (nokome) [inline packages/components/readme.md:45]:
```suggestion
The best reference for how to use and combine these components is our [component library](https://stencila.github.io/designa/).
```

**Review by nokome — COMMENTED** [review]
Thanks Alex, I've made some suggestions. I will merge in a little while, if you do not do so.

**Comment by alex-ketch** [comment]
@nokome I'm fixing up the Storybook failure, and will incorporate your suggestions

**Pull request merged**